build-openjdk11.sh

```diff
@@ -114,7 +114,7 @@ build() {
        flag=""
        dbg_level="$debug"
     fi
-    bash configure \
+    scl enable devtoolset-8 "bash configure \
```

Comment:
We should use `--` as a separator between `scl enable ...` and the actual command to run in the enabled environment.